.github/workflows/build.yml

```
@@ -19,7 +19,7 @@ jobs:
         run: |
           apt-get update
           apt-get install -y git python3.9 pip ninja-build cudnn9-cuda-12
-          pip install cmake==3.21.0
+          pip install cmake==3.21.0 pybind11[global]
```

Now that Ninja is a Python dependency, we should install with Pip instead of apt.
```suggestion
          apt-get install -y git python3.9 pip cudnn9-cuda-12
          pip install cmake==3.21.0 ninja-build pybind11[global]
```
We may also want to consider using the cuDNN Python package as well.